t.py

```diff
@@ -0,0 +1,2 @@
+import json
```

Comment:
`black`: this file needs to be formatted  <!-- Automatically posted by Lintly -->